rover/start/start_can.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+source $HOME/ros_can_nodes/devel/setup.bash
+sudo slcand -o -c -s6 /dev/serial/by-id/usb-CANtact_CANtact_dev_00000000001A-if00 can0
```

Comment:
I think the actual full path of this can change. Wouldn't it be better to use `/etc/network/interfaces` or a udev rule for this?

Reply:
@simonireland would you happen to know how to do this?

Reply:
We have decided to look at that later after the competition